src/devtools/client/debugger/src/components/PrimaryPanes/SourcesTree.tsx

```diff
@@ -46,13 +45,12 @@ import { Source } from "../../reducers/sources";
 
 type $FixTypeLater = any;
 
-function shouldAutoExpand(depth: number, item: TreeNode, debuggeeUrl: string) {
+function shouldAutoExpand(depth: number, item: TreeNode) {
   if (depth !== 1) {
     return false;
   }
 
-  const { host } = debuggeeUrl.length ? parse(debuggeeUrl) : { host: "" };
-  return item.name === host;
```

Comment:
:hand: I am actually curious about this change. I think the intent was to expand the root tree node representing "the real app source" - any reason this changed?

Reply:
@jcmorrow correct - i think CONNECT is firefox devtools legacy and thus dead